cabby/geo/walk.py

```diff
@@ -36,10 +36,49 @@
 
 _Geo_DataFrame_Driver = "GPKG"
 OSM_CRS = 32633  # UTM Zones (North).
+SEED = 4
+POI_DISTANCE = 50
 
 
-def compute_route(start_point: Point, end_point: Point, graph: nx.MultiDiGraph,
-          nodes: GeoDataFrame) -> Optional[GeoDataFrame]:
+def compute_route_from_nodes(origin_id: int, 
+                              goal_id: int, 
+                              graph: nx.MultiDiGraph,
+                              nodes: GeoDataFrame) -> Optional[GeoDataFrame]:
+  '''Returns the shortest path between a starting and end point.
+  Arguments:
+  origin_id: The node id of the origin point.
```

Comment:
Nit: indent the list of Args and Returns

Reply:
fixed indentation